.meteor/packages

```diff
@@ -45,3 +45,5 @@ dispatch:mocha-phantomjs
 xolvio:cleaner
 reactive-var
 juliancwirko:s-alert
+panter:qrcode
+juliancwirko:s-alert
```

Comment:
@singhsanket143 it's duplicated

Reply:
@peter-hank I have now rectified this one also